BlueprintUICommonControls/Tests/Sources/AttributedLabelTests.swift

```diff
@@ -18,8 +18,26 @@ class AttributedLabelTests: XCTestCase {
         compareSnapshot(of: element)
         
     }
+    
+    func test_numberOfLines_sizing() {
```

Comment:
Definitely need to expand these tests for all the various cases we've found within SPOS.